src/Microsoft.ML.Transforms/MissingValueHandlingTransformer.cs

```diff
@@ -131,15 +132,15 @@ public static IDataView Create(IHostEnvironment env, IDataView input, string out
         }
 
         /// Factory method for SignatureDataTransform.
-        internal static IDataTransform Create(IHostEnvironment env, Arguments args, IDataView input)
+        internal static IDataTransform Create(IHostEnvironment env, Options args, IDataView input)
```

Comment:
>internal [](start = 8, length = 8)

can this be private too?

Reply:
Sure I can fix this, I will need to fix the entrypoints in NAHandling as well then.

---
In reply to: [253124651](https://github.com/dotnet/machinelearning/pull/2363#discussion_r253124651) [](ancestors = 253124651)

Reply:
Actually it's not that easy, if it's not a problem I'd rather keep it internal.

---
In reply to: [253165971](https://github.com/dotnet/machinelearning/pull/2363#discussion_r253165971) [](ancestors = 253165971,253124651)